frontend/packages/types/src/shared.ts

```diff
@@ -16,6 +16,8 @@ export class PercentageSet {
 }
 
 export interface Percentage {
+  kind?: string
```

Comment:
Why is this added, Only model is using this data.
If we are using, than we should probably create another interface.